extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii

```diff
@@ -0,0 +1,74 @@
+tags: {  # amp-google-read-aloud-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-google-read-aloud-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-google-read-aloud-player>
+  html_format: AMP
+  tag_name: "AMP-GOOGLE-READ-ALOUD-PLAYER"
+  requires_extension: "amp-google-read-aloud-player"
+  attrs: {
+    name: "data-api-key"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-tracking-ids"
+    mandatory: true;
+    value_regex_casei: "^UA-\\d+-\\d+(\\s*,\\s*UA-\\d+-\\d+)*";
+  }
+  attrs: {
+    name: "data-voice"
+    mandatory: true;
+  }
+  attrs: {
+    name: "data-url"
+    value_url {
+      protocol: 'https';
+      allow_relative: false;
+    }
+  }
+  attrs: {
+    name: "data-speakable"
+    value: ""
+  }
+  attrs: {
+    name: "data-call-to-action-label"
+  }
+  attrs: {
+    name: "data-locale"
+  }
+  attrs: {
+    name: "data-intro"
+    value_url {
+      protocol: 'https';
+      allow_relative: false;
+    }
+  }
+  attrs: {
+    name: "data-outro"
+    value_url {
+      protocol: 'https';
+      protocol: 'http';
+      allow_relative: false;
+    }
+  }
+  attrs: {
+    name: "data-ad-tag-url"
+    value_url {
+      protocol: 'https';
+      protocol: 'http';
+      allow_relative: false;
+    }
+  }
+  spec_url: "https://amp.dev/documentation/components/amp-google-read-aloud-player"
```

Comment:
Would it be a good idea to refer to documentation for Tracking ID from the component documentation? E.g. to this: https://support.google.com/analytics/answer/7372977?hl=en